packages/ckeditor5-ui/src/editorui/poweredby.ts

```diff
@@ -109,11 +109,17 @@ class PoweredByView extends View<HTMLDivElement> {
 			isColorInherited: false
 		} );
 
+		// Get the base font size to calculate the icon size in `em` units.
+		const baseFontSize = parseInt( getComputedStyle( document.documentElement ).getPropertyValue( '--ck-font-size-base' ), 10 );
+		// Calculate the icon size in `em` units.
+		const iconWidthInEm = `${ ICON_WIDTH / baseFontSize }em`;
+		const iconHeightInEm = `${ ICON_HEIGHT / baseFontSize }em`;
```

Comment:
Maybe we could include this in the stylesheet?

Reply:
You mean already calculated value in `em` units?

Reply:
Addressed in 65028dc.